spec/libsass-closed-issues/issue_2786.hrx

```diff
@@ -0,0 +1,19 @@
+<===> input.scss
+@supports (foo: bar) or {
+}
+
+<===> status
+65
+
+<===> error
+Error: Invalid CSS after "... (foo: bar) or ": expected @supports condition (e.g. (display: flexbox)), was "{"
+        on line 1:24 of /sass/spec/libsass-issues/issue_2786/input.scss
+>> @supports (foo: bar) or {
+
+   -----------------------^
+
+<===> error-dart-sass
```

Comment:
The style guide says "use dart sass output as canonical" but here Ruby and libsass have a much better error message. I've left a question on the style guide about this.